assessments/projects/opa/README.md

```diff
@@ -0,0 +1,111 @@
+# OPA Security Assessment
+
+Completed: September 2019
+
+Security reviewers: Justin Cappos, Justin Cormack, Brandon Lum, Sarah Allen
+
+Project security lead: Ash Narkar
+
+* Source code: [github.com/open-policy-agent/opa](https://github.com/open-policy-agent/opa)
+* Web site: [openpolicyagent.org](https://www.openpolicyagent.org/)
+
+## Background
+
+Deployed cloud native applications have policies that must be correctly enforced
+for security reasons.  The desired overall policy may span many different pieces
+of software and configurations.  Open Policy Agent (OPA) seeks to consolidate
+and control policy configuration in a consistent manner across the different
+components of a cloud native deployment.
+
+_Maturity_
+
+The core technology is in production with a broad set of companies, including
+Netflix, Cloudflare, Chef, and others ([ADOPTERS.md](https://github.com/open-policy-agent/opa/blob/master/ADOPTERS.md)).
+Code contributions are primarily from Styra, yet there has been community participation
+from a wide range of adopters ([77 contributors](https://github.com/open-policy-agent/opa/graphs/contributors),
+with Chef engineer in top 4 and Google engineer co-authoring RegoV2 specification).
+
+## Summary
+
+**Design**: The goal of OPA is to centralize and abstract out policy
+functionality to make it easier to manage policy holistically.
+Policies are written in a custom declarative language (“Rego”) which is read by
+OPA agents that are queried by the software performing checks.  The OPA agent
+may return rich information about the status of a policy check.
+
+**Analysis**: There is a clear benefit for adopters who have heterogeneous
+infrastructure or high rate of change where lack of policy enforcement would
+present significant business risk. The added complexity of OPA is not trivial.
+OPA enables “policy as code” and Rego policy expressions require the same care
+to develop as any security critical code. OPA reduces risk by isolating policy
+from business logic. Concise declarative policy has the potential to support
+efficient review; however, that relies upon a reviewer who fully understands the
+syntax and tools for effective validation. Hence usability and security around
+Rego and interaction between aspects of OPA becomes more critical.
+
+All questions from reviewers were addressed in [self-assessment](self-assessment.md)
+with non-critical issues captured as issues and noted below.
+
+## Recommendations
+
+Rego has a non-trivial learning curve and even in demos shows there is potential
+for confusion. Examining the language’s potential for confusion and mitigating
+situations where these problems arise would help to improve user security.
+
+### Project recommendations
```

Comment:
suggestion from @dshaw @pragashj -- for clarification, let's rename:  "Recommendations to the project team"

Reply:
Done